tests/foreman/ui/test_location.py

```diff
@@ -147,6 +148,7 @@ def test_positive_update_user(session):
         assert user.login in loc_values['users']['resources']['unassigned']
 
 
+@skip_if_bug_open('bugzilla', 1730292)
```

Comment:
Closed as dupe of another bug which is WONTFIX

Reply:
yeah .. was checking the same ... looks modify the tests now :)

Reply:
@ntkathole @omkarkhatavkar tests are removed and updated the same in PR too ... would ask a revisit again on it ...
I am resolving this one no more applicable.